adapters/handlers/grpc/v1/parse_search_request_test.go

```diff
@@ -253,6 +253,105 @@ func TestGRPCRequest(t *testing.T) {
 			},
 			error: false,
 		},
+		{
+			name: "hybrid nearvector returns all named vectors with fp32 vectors",
```

Comment:
could you add these tests also for non-hybrid near vector?

Reply:
yes, added tests here too